js/client-chat-tournament.js

```diff
@@ -251,7 +251,11 @@
 				case 'create':
 					var format = Tools.getEffect(data[0]).name;
 					var type = data[1];
-					this.room.$chat.append("<div class=\"notice tournament-message-create\">A " + Tools.escapeHTML(format) + " " + Tools.escapeHTML(type) + " Tournament has been created.</div>");
+					if ("aeiou".includes(format.charAt(0).toLowerCase())) {
```

Comment:
The thing I don't like about this is that `A UU Tournament...` sounds better than `An UU Tournament...` imo

it might just be my personal preference, but grammatically correct doesnt mean it always has to be used, which is what you're aiming for here.